incubator/airflow/README.md

```diff
@@ -0,0 +1,127 @@
+# Airflow / Celery
+
+[Airflow](https://airflow.apache.org/) is a platform to programmatically author, schedule and
+monitor workflows.
+
+
+## Install Chart
+
+To install the Airflow Chart into your Kubernetes cluster :
+
+```bash
+helm install --namespace "airflow" --name "airflow" incubator/airflow
+```
+
+After installation succeeds, you can get a status of Chart
+
+```bash
+helm status "airflow"
+```
+
+If you want to delete your Chart, use this command:
+
+```bash
+helm delete  --purge "airflow"
+```
+
+### Helm ingresses
+
+The Chart provides ingress configuration to allow customization the installation by adapting
+the `config.yaml` depending on your setup. Please read the comments in the value.yaml file for more
+detail on how to configure your reverse proxy or load balancer.
+
+### Prefix
+
+This Helm automatically prefixes all names using the release name to avoid collisions.
+
+### URL prefix
+
+This chart exposes 2 endpoints:
+
+- Airflow Web UI
+- Flower, a debug UI for Celery
+
+Both can be placed either at the root of a domain or at a sub path, for example:
+
+```
+http://mycompany.com/airflow/
+http://mycompany.com/airflow/flower
+```
+
+NOTE: Mounting the Airflow UI under a subpath requires an airflow version > 1.9.x. For the moment
+(March 2018) this is **not** available on official package, you will have to use an image where
+airflow has been updated to its current HEAD. You can use the following one:
+`stibbons31/docker-airflow-dev:2.0dev`
+
+Please also note than Airflow UI and Flower do not behave the same:
+
+- Airflow Web UI behave transparently, to configure it one just need to specify the
+  `ingress.web.path` value.
+- Flower cannot handle this scheme directly and requires to use an URL rewrite mechanism in front
+  of it. In short, it is able to generate the right URLs in the returned HTML file but cannot
+  respond to these URL. It is commonly found in software that wasn't intended to work under
+  something else than a root URL or localhost port. To use it, see the `value.yaml` in detail on how
+  to configure your ingress controller to rewrite the URL (or "strip" the prefix path).
+
+### Airflow configuration
+
+`airflow.cfg` configuration can be changed by defining environment variables in the following form:
+`AIRFLOW__<section>__<key>`.
+
+See the
+[Airflow documentation for more information](http://airflow.readthedocs.io/en/latest/configuration.html?highlight=__CORE__#setting-configuration-options)
+
+This helm chart allows you to add these additional settings with the value key `airflow.config`.
+
+### Worker Statefulset
+
+Celery workers uses StatefulSet instead of deployment.
+It is used to freeze their DNS using a Kubernetes Headless Service, and allow the webserver to
+requests the logs from each workers individually.
+This requires to expose a port (8793) and ensure the pod DNS is accessible to the web server pod,
+which is why StatefulSet is for.
+
+## DAGs Deployment
+
+### Mount a Persistent Volume
```

Comment:
Can you add some more instructions here on how to put DAGs on a persistent volume?

I assume you set `.persistence.enabled` to `true`, but what then? How do I put my DAGs on the volume and where do they go?

Reply:
See https://github.com/kubernetes/charts/pull/3959/files#diff-01d23035d5d39e3bc452aad4b56fdc90R165

Reply:
@Jdban You can see in the template for the web/scheduler/etc Pods how the claim is referenced: https://github.com/gsemet/charts/blob/airflow/incubator/airflow/templates/deployments-web.yaml#L79

So this should be enabled in your values.yaml like this:
```
  persistence:
    enabled: true
    existingClaim: pvc-airflow-dags
    accessMode: ReadWriteMany
    storageClass: azurefile
    size: 1Gi
```